api/src/pcapi/routes/serialization/finance_serialize.py

```diff
@@ -53,3 +64,65 @@ def from_orm(cls, invoice: finance_models.Invoice) -> "InvoiceResponseModel":
 
 class InvoiceListResponseModel(BaseModel):
     __root__: list[InvoiceResponseModel]
+
+
+class LinkedVenues(BaseModel):
+    """A venue that is already linked to a bank account."""
+
+    id: int
+    common_name: str
+
+    class Config:
+        orm_mode = True
+
+
+class ManagedVenues(BaseModel):
+    id: int
+    common_name: str
+    siret: str
+    bankAccountId: int | None
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏On a un `ConfiguredBaseModel` à côté du `BaseModel`
Par défaut il a un alias_generator qui fait snake<->camel

On peut préférer ça dans ces classes pydantic je trouve, pour éviter le mélange serpent/chameau
Idéalement, dans le code api, tout est snake

Enfin [low logaf](https://blog.danlew.net/2020/04/15/the-logaf-scale/) là dessus, ça peut venir après

Reply:
Merci pour le tip, je ne connaissais pas ce model ! Je vais juste corriger la typo (common_name) pour rester uniforme avec le reste de ma PR (d’ailleurs je remarque que même les models qui héritent de `ConfiguredBaseModel` utilise quand même la camelCase